custom_components/elasticsearch/es_doc_creator.py

```diff
@@ -29,6 +29,17 @@
 from custom_components.elasticsearch.system_info import SystemInfo
 
 ALLOWED_ATTRIBUTE_TYPES = tuple | dict | set | list | int | float | bool | str | None
+SKIP_ATTRIBUTES = [
+    "friendly_name",
+    "id",
+    "type",
```

Comment:
What sort of data are you seeing in the `type` attribute? I don't have this one, so I'm not familiar with its contents.

Reply:
yeah maybe type should stay